docs/framework/wcf/feature-details/unsupported-scenarios.md

```diff
@@ -3,110 +3,112 @@ title: "Unsupported Scenarios"
+
+- Do a `p/invoke` of `CertGetCertificateContextProperty`, and inspect `dwProvType` on the returned `CertGetCertificateContextProperty`.
+
+- Use the  `certutil` command from the command line for querying certificates. For more information, see [Certutil tasks for troubleshooting certificates](/previous-versions/orphan-topics/ws.10/cc772619(v=ws.10)).
+
+## Message Security Fails if Using ASP.NET Impersonation and ASP.NET Compatibility Is Required
+
+WCF does not support the following combination of settings because they can prevent client authentication from occurring:
+
+- ASP.NET Impersonation is enabled. This is done in the Web.config file by setting the `impersonate` attribute of the <`identity`> element to `true`.
+
+- ASP.NET compatibility mode is enabled by setting the `aspNetCompatibilityEnabled` attribute of the [\<serviceHostingEnvironment>](../../configure-apps/file-schema/wcf/servicehostingenvironment.md) to `true`.
+
+- Message mode security is used.
+
+The work-around is to turn off the ASP.NET compatibility mode. Or, if the ASP.NET compatibility mode is required, disable the ASP.NET impersonation feature and use WCF-provided impersonation instead. For more information, see [Delegation and Impersonation](delegation-and-impersonation-with-wcf.md).
+
+## IPv6 Literal Address Failure
+ Security requests fail when the client and service are on the same machine, and IPv6 literal addresses are used for the service.
+
+ Literal IPv6 addresses work if the service and client are on different machines.
+
+## WSDL Retrieval Failures with Federated Trust
+ WCF requires exactly one WSDL document for each node in the federated trust chain. Be careful not to set up a loop when specifying endpoints. One way in which loops can arise is using a WSDL download of federated trust chains with two or more links in the same WSDL document. A common scenario that can produce this problem is a federated service where the Security Token Server and the service are contained inside the same ServiceHost.
+
+ An example of this situation is a service with the following three endpoint addresses:
+
+- `http://localhost/CalculatorService/service` (the service)
+
+- `http://localhost/CalculatorService/issue_ticket` (the STS)
+
+- `http://localhost/CalculatorService/mex` (the metadata endpoint)
+
+ This throws an exception.
+
+ You can make this scenario work by putting the `issue_ticket` endpoint elsewhere.
+
+## WSDL Import Attributes can be Lost
+ WCF loses track of the attributes on a `<wst:Claims>` element in an `RST` template when doing a WSDL import. This happens during a WSDL import if you specify `<Claims>` directly in `WSFederationHttpBinding.Security.Message.TokenRequestParameters` or `IssuedSecurityTokenRequestParameters.AdditionalRequestParameters` instead of using the claim type collections directly.  Since the import loses the attributes, the binding does not round trip properly through WSDL and hence is incorrect on the client side.
```

Comment:
```suggestion

WCF loses track of the attributes on a `<wst:Claims>` element in an `RST` template when doing a WSDL import. This happens during a WSDL import if you specify `<Claims>` directly in `WSFederationHttpBinding.Security.Message.TokenRequestParameters` or `IssuedSecurityTokenRequestParameters.AdditionalRequestParameters` instead of using the claim type collections directly.  Since the import loses the attributes, the binding does not round trip properly through WSDL and hence is incorrect on the client side.
```